<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KMediaWidgetPrivate</class>
 <widget class="QWidget" name="KMediaWidgetPrivate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>575</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KMediaWidget</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-multimedia">
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="w_player" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="w_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="w_play">
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QDial" name="w_volume">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QPushButton" name="w_menu">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="show-menu">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="7">
       <widget class="QLabel" name="w_status">
        <property name="text">
         <string/>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="w_previous">
        <property name="text">
         <string>Previous</string>
        </property>
        <property name="icon">
         <iconset theme="go-previous-view">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="w_next">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset theme="go-next-view">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSlider" name="w_position">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
